pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -29,13 +29,13 @@
 import torch
 import yaml
 
-from pytorch_lightning import _logger as log
 from pytorch_lightning.callbacks.base import Callback
 from pytorch_lightning.utilities import rank_zero_info, rank_zero_only, rank_zero_warn
 from pytorch_lightning.utilities.cloud_io import get_filesystem
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from pytorch_lightning.utilities.warnings import WarningCache
 
+log = logging.getLogger(__name__)
```

Comment:
can we rather have it protected so no one will load it from here?

Reply:
this will be a lot of changes renaming all `log` to `_log`. Are you sure I should do this?